app/api/v1/student/parkingSpotRequest/route.ts

```diff
@@ -56,11 +49,13 @@ export async function PUT(request: Request) {
   const newJson = await request.json();
   const user = await currentUser();
   const dbUser = await User.findOne({
-    email: user?.emailAddresses[0].emailAddress,
+    clerkUserId: user?.id
   });
 
-  // Prevent the user from directly setting the submitted value
+  // Prevent the user from directly setting the submitted value or decision status
   newJson.submitted = false;
+  newJson.decicion = 'undecided';
+  newJson.user = dbUser._id;
```

Comment:
This would also reset the status whenever the user updates their form, if that's relevant at all.